TvMaterialCatalog/app/src/main/java/com/google/tv/material/catalog/AppBar.kt

```diff
@@ -129,7 +131,10 @@ private fun Actions(
         )
     )
 
-    Row(horizontalArrangement = Arrangement.spacedBy(16.dp)) {
+    Row(
+        horizontalArrangement = Arrangement.spacedBy(16.dp),
+        modifier = Modifier.windowInsetsPadding(WindowInsets(0.dp, 0.dp, 8.dp, 0.dp))
```

Comment:
Out of curiosity, does window insets affect TV devices? I assumed it only affects mobile devices.

Reply:
Good call. I think this should just be `Modifier.padding()`